R/pkg/R/generics.R

```diff
@@ -771,6 +771,10 @@ setGeneric("over", function(x, window) { standardGeneric("over") })
 
 ###################### WindowSpec Methods ##########################
 
+#' @rdname orderBy
+#' @export
```

Comment:
hmm, that's interesting. I'm not sure if this is necessary, or if this works. I think the purpose of setGeneric would only need this once. By putting a @rdname tag it will show up on the Rd file, but I don't think we need to put it twice, once in different Rd?